changelogs/unreleased/7556-prevent-project-deletion-if-environment.yml

```diff
@@ -0,0 +1,8 @@
+---
+description: Prevent the deletion of a project if this project contains one or multiple environments attached to it.
+issue-nr: 7556
+issue-repo: inmanta-core
+change-type: minor
+destination-branches: [master, iso7]
+sections:
+  feature: "{{description}}"
```

Comment:
The `feature` section is intended for big new features. This isn't. So we should either move it to `minor-improvement` if we consider this new functionality or to `bugfix` if we consider the old behavior to be a bug. I am fine with both.